client/my-sites/media-library/test/data-source.jsx

```diff
@@ -53,8 +54,8 @@ describe( 'MediaLibraryDataSource', () => {
 					/>
 				</ReduxProvider>
 			);
-			expect( wrapper.find( 'button[data-source="google_photos"]' ) ).toHaveLength( 1 );
-			expect( wrapper.find( 'button[data-source="pexels"]' ) ).toHaveLength( 0 );
+			expect( screen.queryByText( 'Google Photos' ) ).toBeInTheDocument();
```

Comment:
The official [recommendation](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence) suggests that we use `query*` screen queries only for checking non-existence (like on the line below).

```suggestion
			expect( screen.getByText( 'Google Photos' ) ).toBeInTheDocument();
```

This applies to at least one more instance in this PR.